files/zh-cn/web/api/web_animations_api/using_the_web_animations_api/index.md

```diff
@@ -272,25 +272,25 @@ var aliceChange = document
   );
 ```
 
-爱丽丝的动画让她的尺寸从一半到 8 秒的两倍。然后我们暂停她：
+爱丽丝的动画让她的尺寸在8秒内从一半到两倍。然后我们暂停她：
```

Comment:
Let's [add space between Chinese words and Latins](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E4%B8%AD%E6%96%87%E5%92%8C%E6%8B%89%E4%B8%81%E8%AF%AD%E7%B3%BB%E6%96%87%E5%AD%97%E9%97%B4%E5%8A%A0%E7%A9%BA%E6%A0%BC)

```suggestion
爱丽丝的动画让她的尺寸在 8 秒内从一半到两倍。然后我们暂停她：
```